docs/recipes/git-show.rst

```diff
@@ -40,6 +40,45 @@ Show all files in commit
     >>> for e in commit.tree:
     >>>     print(e.name)
 
+======================================================================
+Produce something like a ``git show`` message
+======================================================================
+
+In order to display timezone information you have to create a subclass of
+tzinfo as described in the `Python datetime documentation`_::
+
+    from datetime import tzinfo, timedelta
+    class FixedOffset(tzinfo):
+        """Fixed offset in minutes east from UTC."""
+
+        def __init__(self, offset):
+            self.__offset = timedelta(minutes = offset)
+
+        def utcoffset(self, dt):
+            return self.__offset
+
+        def tzname(self, dt):
+            return None
+
+        def dst(self, dt):
+            return timedelta(0) # dealing with DST would be too complicated
```

Comment:
That would be convenient. The `.format` version would be shorter, but also ugly, since you have to deal with the sign. `123 / 60 == 2`, `-123 / 60 == -3`